packages/core/addon/models/bard-request-v2/fragments/base.js

```diff
@@ -5,11 +5,23 @@
 import attr from 'ember-data/attr';
 import Fragment from 'ember-data-model-fragments/fragment';
 import { validator, buildValidations } from 'ember-cp-validations';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+import { assert } from '@ember/debug';
+import { isPresent } from '@ember/utils';
 
 const Validations = buildValidations({
   field: validator('presence', {
     presence: true,
     message: 'The `field` field cannot be empty'
+  }),
+  type: validator('inclusion', {
+    in: ['dimension', 'metric', 'time-dimension'],
```

Comment:
I think the rfc said `timeDimension`, so we should make sure those are in sync

Reply:
I think `time-dimension` is what we should use in this case since the type string is eventually used to lookup the `time-dimension` metadata model in the registry.